site/content@beta/backend-apis/apis.md

```diff
@@ -0,0 +1,276 @@
+<!-- End of description -->
+
+<!-- > The two properties are within the namespace named after your `application id`. If you don't know your `application id`, you can check it with the request below.
+> In this case, `client_id` is your `application id`. You can find the `client_id` in [**Developers Console**](https://developers.livechatinc.com/console/) **>> the Authorization building block** of your app.
+
+```json
+curl https://accounts.livechatinc.com/info -H "Authorization: Bearer c5e4f61e1a6c3b1521b541bc5c5a2ac5"
+{
+    "access_token":"c5e4f61e1a6c3b1521b541bc5c5a2ac5",
+    "client_id":"58737b5829e65621a45d598aa6f2ed8e",
+    ...
+}
+```
+
+> Now, you can set up the properties within the existing chat from the customer's perspective using the Agent/Customer Chat API method: [update_chat_properties](../customer-chat-api/#update-chat-properties).
+
+```json
+curl -v https://api.livechatinc.com/customer/v0.5/action/update_chat_properties \
+    -H "Content-Type: application/json" \
+    -H "Authorization: Bearer c5e4f61e1a6c3b1521b541bc5c5a2ac5" \
+    -X POST -d ' \
+    {
+        "chat_id": "a0c22fdd-fb71-40b5-bfc6-a8a0bc3117f5",
+        "properties": {
+            "58737b5829e65621a45d598aa6f2ed8e": {
+                "rating_score": 10,
+                "rating_comment": "This guy is a support hero, he helped me a lot."
+            }
+        }
+    }'
+```
+
+
+> The properties will be visible from the agent's perspective in the chat object as a return element (in the sample response from the [get_archives](../agent-chat-api/#get-archives) method).
+
+```js
+{
+	"chats": [{
+		"chat": {
+			"id": "a0c22fdd-fb71-40b5-bfc6-a8a0bc3117f5",
+			"users": [
+				// array of "User" objects
+			],
+			"thread": {
+				// "Thread" object
+			},
+            "properties": {
+                "58737b5829e65621a45d598aa6f2ed8e": {
+                    "rating_score": 10,
+                    "rating_comment": "This guy is a support hero, he helped me a lot."
+                }
+                //other namespaces
+            }
+		}
+	}],
+	"pagination": {
+		"page": 1,
+		"total": 3
+	}
+}
+```
+
+> They will also be visible in the [chat_properties_updated](../customer-chat-api/#chat-properties-updated) push.
+
+```js
+{
+   "chat_id": "a0c22fdd-fb71-40b5-bfc6-a8a0bc3117f5",
+"properties": {
+	"58737b5829e65621a45d598aa6f2ed8e": {
+           "rating_score": 10,
+           "rating_comment": "This guy is a support hero, he helped me a lot.",
+       }
+}
+}
+``` -->
+
+## Webhooks
+
+LiveChat provides a number of webhooks. You can manage them via the [Configuration API](../beta-docs/configuration-api/#webhooks).
+We can distinguish **global webhooks** and **bot-specific webhooks**. Once **global webhooks** are set up, you will always receive them. **Bot-specific** webhooks are strongly coupled with the bot's status (`accepting chats`, `not accepting chats`, `offline`). If the bot is offline, webhooks won't be received. 
+
+## Bot Agents
```

Comment:
Is the info on Bot Agents still correct and valid? Or are some parts outdated? Can this section be shorter?